### ISSUE TITLE
@splinetool/loader v0.9.139 is crashing react apps in production mode

### ISSUE THREAD

**Issue by buesing** [splinetool/r3f-spline#11]
Hi, since upgrading to v0.9.139 my next app crashes when exported, with the error
```
Hydration failed because the initial UI does not match what was rendered on the server.
```
v0.9.138 works fine. I can't provide a reproduction right now but maybe this enough for you to go on anyway.

**Comment by far0s** [comment]
Hi, I've had the same issue using v0.9.143, rolled back to 0.9.138 and it works fine.

**Comment by marcofugaro** [comment]
Fixed in `@splinetool/loader` version `0.9.145`
